tests/components/wmspro/test_light.py

```diff
@@ -0,0 +1,205 @@
+
+        await hass.services.async_call(
+            Platform.LIGHT,
+            SERVICE_TURN_ON,
+            {ATTR_ENTITY_ID: entity.entity_id},
+            blocking=True,
+        )
+
+        entity = hass.states.get("light.licht")
+        assert entity is not None
+        assert entity.state == "on"
+        assert entity.attributes["brightness"] >= 1
+        assert len(mock_hub_status_prod_dimmer.mock_calls) == before
+
+    with patch(
+        "wmspro.destination.Destination.refresh",
+        return_value=True,
+    ):
+        before = len(mock_hub_status_prod_dimmer.mock_calls)
+
+        await hass.services.async_call(
+            Platform.LIGHT,
+            SERVICE_TURN_OFF,
+            {ATTR_ENTITY_ID: entity.entity_id},
+            blocking=True,
+        )
+
+        entity = hass.states.get("light.licht")
+        assert entity is not None
+        assert entity.state == "off"
+        assert entity.attributes["brightness"] is None
+        assert len(mock_hub_status_prod_dimmer.mock_calls) == before
+
+
+async def test_light_dimm_on_and_off(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_hub_ping: AsyncMock,
+    mock_hub_configuration_prod: AsyncMock,
+    mock_hub_status_prod_dimmer: AsyncMock,
+    mock_action_call: AsyncMock,
+) -> None:
+    """Test that a light entity is dimmed on and off correctly."""
+    assert await setup_config_entry(hass, mock_config_entry)
+    assert len(mock_hub_ping.mock_calls) == 1
+    assert len(mock_hub_configuration_prod.mock_calls) == 1
+    assert len(mock_hub_status_prod_dimmer.mock_calls) >= 1
+
+    entity = hass.states.get("light.licht")
+    assert entity is not None
+    assert entity.state == "off"
+    assert entity.attributes["brightness"] is None
+
+    with patch(
+        "wmspro.destination.Destination.refresh",
+        return_value=True,
+    ):
+        before = len(mock_hub_status_prod_dimmer.mock_calls)
+
+        await hass.services.async_call(
+            Platform.LIGHT,
+            SERVICE_TURN_ON,
+            {ATTR_ENTITY_ID: entity.entity_id},
+            blocking=True,
+        )
+
+        entity = hass.states.get("light.licht")
+        assert entity is not None
+        assert entity.state == "on"
+        assert entity.attributes["brightness"] >= 1
+        assert len(mock_hub_status_prod_dimmer.mock_calls) == before
+
+    with patch(
+        "wmspro.destination.Destination.refresh",
+        return_value=True,
+    ):
+        before = len(mock_hub_status_prod_dimmer.mock_calls)
+
+        await hass.services.async_call(
+            Platform.LIGHT,
+            SERVICE_TURN_ON,
+            {ATTR_ENTITY_ID: entity.entity_id, ATTR_BRIGHTNESS: 128},
+            blocking=True,
+        )
+
+        entity = hass.states.get("light.licht")
+        assert entity is not None
+        assert entity.state == "on"
+        assert entity.attributes["brightness"] == 128
```

Comment:
```suggestion
        assert entity.state == STATE_ON
        assert entity.attributes[ATTR_BRIGHTNESS] == 128
```
Please apply this to the whole file :)